package.json

```diff
@@ -23,6 +23,8 @@
     "build": "grunt default && gulp default",
     "lint": "eslint src/ && grunt lint && gulp lint",
     "testonly": "./resources/node_test.sh",
+    "testmocha": "mocha --require ts-node/register --recursive 'test/*.ts'",
```

Comment:
I prefer using a *-test.ts notation for mocha tests, so if we ever need any helper files they can live in the directory but won't be run as tests